styles.css

```diff
@@ -1,42 +1,53 @@
 * {
-  color: white;
   list-style: none;
   font-family: "Roboto", sans-serif;
-  font-weight: bold;
 }
 
 body {
   width: 100%;
   background-color: #1c1a19;
   margin: 0;
-  background-image: url("imgs/Group.svg");
-  background-repeat: no-repeat;
-  background-position-x: right;
-  background-position-y: bottom;
 }
 
 header {
   width: 100%;
   height: 8vh;
-  background-color: #3c3c33;
+  background-color: #3c3a39;
   display: flex;
   align-items: center;
   justify-content: space-between;
+  font-size: 18px;
+  color: white;
 }
 
-p {
-  margin-left: 5%;
-  font-size: 18px;
+.grid-container {
+  background-color: white;
+  display: grid;
+  justify-content: center;
+}
+
+ul {
+  padding: 0%;
 }
 
 .burgir {
   height: 12px;
   width: 18px;
-  margin-right: 5%;
+  margin-right: 8%;
+}
+
+.doe {
+  margin-left: 5%;
+  font-weight: bold;
 }
 
 .bio {
   padding: 10vh 3vh;
+  background-image: url("imgs/Group.svg");
+  background-repeat: no-repeat;
```

Comment:
[OPTIONAL] 

- [ ] You have done a great job applying this background image 🤝. However, the image is blocking the content of the web page and this makes visitor that comes to your web page guess what the words could be and this does not help in improving user experience 😃. Could you try fixing them? 

your design:
<img width="167" alt="chrome_W8SJcOR3DR" src="https://user-images.githubusercontent.com/81468675/143660790-0f605896-35f4-4383-8577-ed1ce159643d.png">


Original design:
 
<img width="164" alt="chrome_NgkuYicOZK" src="https://user-images.githubusercontent.com/81468675/143660818-be89d236-7fcc-4e8f-9c96-d8c8ca97511c.png">